backend/tests/unit/models/test_websocket_chain_models.py

```
@@ -0,0 +1,404 @@
+Unit tests for chain-related WebSocket models.
+
+Tests ChainProgressUpdate, StageProgressUpdate and related WebSocket functionality
+for sequential agent chains.
+"""
+
+import pytest
+from pydantic import ValidationError
+from tarsy.models.websocket_models import (
+    ChainProgressUpdate, StageProgressUpdate, ChannelType
+)
+from tests.utils import ModelValidationTester, TestUtils
```

Ruff (0.12.2)

`tests.utils.ModelValidationTester` imported but unused

Remove unused import

(F401)